AutoPull_install.ps1

```diff
@@ -0,0 +1,11 @@
+# make dir  
+mkdir "C:\Program Files (x86)\autopull\"
```

Comment:
will this error if the directory already exists?

Reply:
yes it will I will add a -force option so it does not error  for now ..then add  more eror checking later